ztree-diff.el

```diff
@@ -95,6 +95,8 @@ By default paths starting with dot (like .git) are ignored")
 (defvar-local ztree-diff-wait-message nil
   "Message showing while constructing the diff tree.")
 
+(defvar ztree-diff--ediff-previous-window-configuration nil
```

Comment:
I don't want to flame on this, but this I see as a more "modern" convention, which is certainly not emacs-wide, as tons of build-in packages are not using it (ediff, dired, gnus etc).
Moreover there are a lot of other "conventions", i.e. ztree/myvariable etc. Maybe I'll change it at some point but I see no reason in it right now..
And in this package I would prefer the consistency in naming anyway, evening if conventions are not that convenient.